### PR TITLE
Contribution dates rearranged

### PR DESCRIPTION
**Pull request by WebShells**
Updated & Contribution dates rearranged.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
-Copyright (c) 2015 Aamir Shahzad (aka asakpke), RoshanTech.com
-Copyright (c) 2015 Toni Haryanto (aka yllumi)
-Copyright (c) 2016-2017 Ramkrishna Mondal (aka RamkrishnaMondal)
-Copyright (c) 2016 Rinaldy@dbarber (aka rnld26)
-Copyright (c) 2016-2017 Jorge Colmenarez (aka jlctmaster), frontuari.com
-Copyright (c) 2017-2021 Steve Ireland
-Copyright (c) 2017-2021 objecttothis
-Copyright (c) 2017-2021 odiea
-Copyright (c) 2017-2021 WebShells / Shady Sh
+Copyright (c) 2021-2013 jekkos
```
Comment (daN4cat) [inline LICENSE:3]: I'm afraid this is not the correct approach

**Review by daN4cat — CHANGES_REQUESTED** [review]
Please fix it

File: LICENSE (hunk 1 above)
Comment (WebShells) [reply LICENSE:3]: @daN4cat you mean the format "current year - start year" ? I tried sorting them as "start year - current year"  but the display looked mixed up. It's the only way i found to sort them in descending order to go down from 2021 till 2012.

Comment (daN4cat) [reply LICENSE:3]: Agreed but that's not correct, also no idea why we need to sort this list. However, I'm not fussy about it.

**Pull request merged**